config/routes.rb

```diff
@@ -10,10 +10,10 @@
   end
 
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
```

Comment:
## Your Splash Screen
![image](https://user-images.githubusercontent.com/93039359/221381559-720c82f0-6ffe-4085-851a-b689187b2de3.png)

## Design Splash Screen
![image](https://user-images.githubusercontent.com/93039359/221381654-b24b290f-3f20-442a-8ca3-c53b670e046e.png)


**[Optional]**

- [ ] It would be nice if you follow the design because in there is no button with green color and the buttons are at the bottom of the screen :fearful: Kindly follow the design to make your project look professional and attractive to the viewers of your project especially recruiters :+1: